TestCases/turbulence_models/sa/rae2822/turb_SA_NEG_RAE2822.cfg

```diff
@@ -14,7 +14,7 @@ RESTART_SOL= NO
 MACH_NUMBER= 0.729
 %
 % Angle of attack (degrees, only for compressible flows)
-AOA= 2.31
+AOA= 20.31
```

Comment:
I recon, the case still converges to a reasonable solution with that more aggressive AoA?